SalesforceMagic/Http/HttpClient.cs

```diff
@@ -128,11 +148,26 @@ private void AddRequestBody(ref HttpWebRequest webRequest, HttpRequest request)
         public void Dispose()
         {
             Dispose(true);
+            GC.SuppressFinalize(this);
         }
 
-        public void Dispose(bool safe)
+        protected virtual void Dispose(bool disposing)
         {
-            // TODO: Implement IDisposable
+            if (this.disposed)
+            {
+                return;
+            }
+
+            if (disposing)
+            {
+                // Free any other managed objects here.
+                //
+            }
+
+            // Free any unmanaged objects here.
+            //
```

Comment:
Let's remove these comments and the extra if statement for now. If we need it in the future we can add it back.